### DISCUSSION TITLE
Liste der DPs nach Socket Verlust -> resubscribeStates

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2333] [Q&A]
Sollten hier nicht alle DPs drin stehen, die man in seinem Layout nutzt? -> es wären viel zu wenig.
Oder nur die DPs, die gerade im angezeigten TAB genutzt werden?

![image](https://user-images.githubusercontent.com/84818934/233434271-a1d8cb42-8495-4413-89d3-fc10ebda9f9a.png)

**Comment by Zefau** [comment]
Da sind alle DPs drin, die geladen wurden. Also von Tabs, die durch den User aufgerufen wurden.
